appveyor.yml

```diff
@@ -35,6 +34,5 @@ build:
     verbosity: normal
 
 test_script:
-    - cp libzmq/bin/libzmq*.dll Build/bin/%configuration%/
```

Comment:
It was creating a problem because the folder was not there (if I get it correctly, `%configuration%` refers to the Visual Studio MS build) and it was failing the test.

I don't have any experience at all with AppVeyor, look forward your comments on how to fix this or it is ok like this (I don't know why it was needed and/or if it breaks anything in terms of AppVeyor build caches).

Reply:
@gummif at least now, without this copy, all tests passed! 🎉

Reply:
I find it strange that no tests are being run (see No tests were found!!! in the logs), it has been this way for at least a year, so that it unrelated to this PR.

Reply:
Good catch, you're right! I will give it a look. I will open a dedicated PR for that, it is not strictly related to this one, but since this PR changes the test part, I think it would be worth fixing it before, so we can check that everything works also on Windows platform.

Reply:
I wouldn't worry too much about it, I would like to move to github actions soon instead of travis/appveyor which should make managing the CI easier.

Reply:
So, if you want to skip this problem on the windows CI, feel free to review the PR. On my side it should be ok.